### PR TITLE
Fix RabbitMq console

### PR DESCRIPTION
**Pull request by TomPradat**

### CODE REVIEW

File: src/Router/RabbitMq/Adapter.php [hunk 1]
```diff
@@ -32,7 +32,11 @@ public function __invoke(Message $message)
             'date' => $message->exchange,
         ];
 
-        $dataModel = new DataModel($type, $data, $metadata);
+        if (null === ($arrayData = json_decode($data, true))) {
```
Comment (Th3Mouk) [inline src/Router/RabbitMq/Adapter.php]:
```php
if (null === ($data = json_decode($message->getData(), true))) {
     $data = $message->getData();
}
```

File: src/Router/RabbitMq/Adapter.php [hunk 2]
```diff
@@ -32,7 +32,11 @@ public function __invoke(Message $message)
...
+            throw new \Exception('Cannot decode data');
+        }
+
+        $dataModel = new DataModel($type, $arrayData, $metadata);
```
Comment (Th3Mouk) [inline src/Router/RabbitMq/Adapter.php]: $arrayData -> $data

**Comment by Th3Mouk** [comment]
IMO this must be 3 different commits because they address different problems.

**Comment by TomPradat** [comment]
> IMO this must be 3 different commits because they address different problems.

Done

**Comment by Th3Mouk** [comment]
CI is broken and must be fixed

**Pull request closed without merging**